src/components/Portfolio/PaidAmountModal.vue

```diff
@@ -0,0 +1,66 @@
+<template>
+	<div>
+		<span
+			class="tw-text-link tw-cursor-pointer"
+			@click="showModal = true"
+		>
+			${{ formattedAmount }} repaid to you
+		</span>
+
+		<kv-lightbox
+			:visible="showModal"
+			@lightbox-closed="showModal = false"
+			title="Payment history"
+		>
+			<div class="tw-px-4">
+				<div class="tw-grid tw-grid-cols-2">
+					<div v-for="(payment, index) in paymentHistory" :key="index" class="tw-contents">
+						<span>{{ payment.date }}:</span>
+						<span class="tw-relative">
+							<span v-if="payment.type === 'loss'" class="tw-absolute tw-right-full">-</span>
+							${{ payment.amount }}
+							{{ payment.type === 'loss' ? 'currency loss' : 'repaid' }}
+						</span>
+					</div>
+				</div>
+			</div>
+		</kv-lightbox>
+	</div>
+</template>
+
+<script>
+import numeral from 'numeral';
+import { KvLightbox } from '@kiva/kv-components';
+
+export default {
+	name: 'PaidAmountModal',
+	components: {
+		KvLightbox
+	},
+	props: {
+		amount: {
+			type: [Number, String],
```

Comment:
Is this prop really two possible types?

Reply:
ah no, I was testing both types and meant to change this back.  should be ok now.